enterprisemediawiki/meza
# Upload wizard

This pull request adds [Extension:UploadWizard](https://www.mediawiki.org/wiki/Extension:UploadWizard) to the list of extensions installed.

## Testing

1. `curl -LO https://raw.githubusercontent.com/enterprisemediawiki/meza/upload-wizard/scripts/install.sh`
2. `sudo bash install.sh`
3. Use `upload-wizard` branch
4. Verify uploading multiple files with Upload Wizard
